<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>275</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Crear Cl√≠nica</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="volverButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>210</y>
      <width>131</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Volver</string>
    </property>
   </widget>
   <widget class="QPushButton" name="aceptarButton">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>200</y>
      <width>211</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Aceptar</string>
    </property>
   </widget>
   <widget class="QWidget" name="formLayoutWidget">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>30</y>
      <width>721</width>
      <height>139</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="formLayout">
     <property name="labelAlignment">
      <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
     </property>
     <property name="formAlignment">
      <set>Qt::AlignmentFlag::AlignCenter</set>
     </property>
     <item row="0" column="0">
      <widget class="QLabel" name="idLabel">
       <property name="text">
        <string>ID:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="idInput"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="nameLabel">
       <property name="text">
        <string>Nombre:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="nameInput"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="municipioLabel">
       <property name="text">
        <string>Municipio:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="municipioInput"/>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="provinciaLabel">
       <property name="text">
        <string>Provincia:</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="provinciaInput"/>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
